daemon/config/config.go

```diff
@@ -20,6 +20,13 @@ type Config struct {
 	// ListenCRI is the listening address which serves CRI.
 	ListenCRI string
 
+	// StreamServerAddress is the ip address streaming server of CRI is listening on.
+	// The default host interface is used if not specified.
+	StreamServerAddress string
```

Comment:
Could we encapsulate a CRI config struct to keep all configs about CRI implementation?

Reply:
@allencloud  Yes, it is a good idea :)